tests/test_views.py

```diff
@@ -1,21 +1,87 @@
 """
-Tests for the frontend.
+Unit tests for the frontend code.
 """
 
 import unittest
 
-import ga4gh.server as server
+import ga4gh.backends
 import ga4gh.cli
```

Comment:
This isn't being used, please delete it. I've been using flake8 to catch these kind of problems, it's pretty good.